CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Move Py_DecodeLocale from sysmodule to fileutils. [#1887](https://github.com/PyO3/pyo3/pull/1887)
 - Deprecate `PySys_AddWarnOption`, `PySys_AddWarnOptionUnicode` and `PySys_HasWarnOptions`. [#1887](https://github.com/PyO3/pyo3/pull/1887)
 - Remove function PyTuple_ClearFreeList from python 3.9 above. [#1887](https://github.com/PyO3/pyo3/pull/1887)
+- Replace `unwrap` with `?` in derivation body for enums. [#1888](https://github.com/PyO3/pyo3/pull/1888)
```

Comment:
The user-facing experience is probably "Fixed: panic inside generated `#[derive(FromPyObject)]` for enums"

- I'll push a commit to tweak this shortly and then merge!